site/src/examples/low-barrel/index.js

```diff
@@ -62,19 +62,10 @@
 
 }(d3, fc));
 
-(function(d3, fc) {
-    'use strict';
-
-    var dataGenerator = fc.data.random.financial()
-        .startDate(new Date(2014, 1, 1));
-
-    var container = d3.select('#low-barrel')
-        .datum(dataGenerator(250))
-        .layout();
+(function (d3, fc) {
```

Comment:
Do you need two wrap the components in functions? I am guessing this was left over from when they were in separate files?